src/gui/sdlmain.cpp

```diff
@@ -2454,6 +2459,9 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	Pbool = sdl_sec->Add_bool("rawmouse", Property::Changeable::Always, false);
+	Pbool->Set_help("Mouse is routed through OS API by default. Enable this to poll mouse directly which bypasses OS mouse settings.");
+
```

Comment:
Use simply `always` instead of `Property::Changeable::Always` (necessary `constexpr` is already defined here, we're slowly transitioning away from silly "Property::Changeable" construct.

Change default value to `true` as that is default picked by SDL2 project, I believe for a good reason.

Not sure if `rawmouse` is the best name here - the value that is being changed is "raw input for relative mouse events", but users shouldn't need to know about relative part… `sdl` section have quite long prop names anyway - let's change it to `raw_mouse_input` - will be easier for new users to understand (and less gore'y ;)).

About description: update it accordingly to a new default. For new/updated descriptions we are trying to keep them wrapped somewhere around 80-85 columns for a newly generated .conf file (this is another WIP work, hopefully, we'll manage to make the whole generated .conf file wrapped around 80 columns eventually).

Reply:
I disagree with setting with setting default to true, the whole point of this change is to restore mouse behavior observed in older DosBox version.